### DISCUSSION TITLE
Thorium update

### DISCUSSION THREAD

**Discussion by arinoki** [Alex313031/thorium#483] [General]
So previous version was released almost 3 months ago - any ETA for new one? And thanks for ur work :)

**Comment by Alex313031** [comment]
@arinoki Literally just worked on M119 today!

I will have a blog post explaining where I was, and also addressing the whole easter egg/titus tech drama.

**Comment by arinoki** [comment]
@Alex313031 yeah, i saw this by commits. Thing is - latest released version is m117 - so where 118? 
And any chance of release before new year? ;)

**Comment by Rocman76** [comment]
Thanks @Alex313031 , be great to hear from you and I'm looking forward to the new version

**Comment by Alex313031** [comment]
@arinoki @Rocman76 Because I was gone so long, and because Supermium released M119 with some useful fixes, I decided to skip M118 for all platforms/all releases, and just go straight to M119 stable.

**Comment by proofy** [comment]
Happy new year Alex. Thanx 4 your work. The Linux Tier membership at patreon is missing.;) So may be everybody who need faster updates can  help over patreon $)

**Comment by Alex313031** [comment]
@proofy I can add a non-win7 tier to cover all other platforms, for probably the same 5$ per release (I will never do it "per month" because what if for some reason I don't make a release for a given month, then you just spent money on nothing)

**Comment by Alex313031** [comment]
@proofy @arinoki @validMyID I made a new "Linux" tier that is the same 5$ per release > https://www.patreon.com/ThoriumDeveloper

**Reply by proofy** [reply]
Done

**Reply by validMyID** [reply]
im away from civilisation .... at middle now-where-land in my country.. soon. interesting in win7 actually ^_^
